src/unistd.rs

```diff
@@ -1324,6 +1324,85 @@ pub fn pause() {
     unsafe { libc::pause() };
 }
 
+pub mod alarm {
+    //! Alarm signal scheduling.
+    //!
+    //! See also [alarm](http://pubs.opengroup.org/onlinepubs/9699919799/functions/alarm.html).
```

Comment:
Change the link to "alarm(2)" and move this into its own "# References" section after the Examples section.